src/test/helpers.ts

```diff
@@ -313,6 +313,10 @@ export async function clear(prisma: PrismaClient): Promise<void> {
   await prisma.collectionPartnership.deleteMany({});
   await prisma.collectionPartner.deleteMany({});
 
+  // labels
+  await prisma.collectionLabel.deleteMany({});
+  await prisma.label.deleteMany({});
+
```

Comment:
This needed to be added so that the database was fully emptied between integration test suites.